site/app/entities/email/EmailEntity.php

```diff
@@ -53,6 +56,13 @@ public function getUserId(): string {
         return $this->user_id;
     }
 
+    /**
+     * @return string
+     */
+    public function getToName(): string {
```

Comment:
There's no need to put a redundant PHPDoc comment given that you're not providing any additional information